iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ValidationUtil.java

```diff
@@ -585,9 +585,8 @@ public static void checkConnected(Validable validable, Boolean connected) {
     }
 
     public static void checkPermanentLimit(Validable validable, double permanentLimit) {
-        // TODO: if (Double.isNaN(permanentLimit) || permanentLimit <= 0) {
         if (permanentLimit <= 0) {
-            throw new ValidationException(validable, "permanent limit must be defined and be > 0");
+            throw new ValidationException(validable, "permanent limit must be > 0");
```

Comment:
Now, you must rebase your PR on #2816 and add the same kind of treatment as what you have done for CGMES conversion for XIIDM/JIIDM version under version 1.12.